docs/pages/enroll-resources/database-access/enroll-managed-databases/mongodb-atlas.mdx

```diff
@@ -229,6 +138,54 @@ it exclusively supports authentication using AWS IAM roles.
 </TabItem>
 </Tabs>
 
+## Step 2/4. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+(!docs/pages/includes/database-access/mongodb-required-database-access.mdx!)
+
+## Step 3/4. Set up the Teleport Database Service
+
+(!docs/pages/includes/tctl-token.mdx serviceName="Database" tokenType="db" tokenFile="/tmp/token"!)
+
+Install and configure Teleport where you will run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Server">
+
+(!docs/pages/includes/install-linux.mdx!)
+
+(!docs/pages/includes/database-access/db-configure-start.mdx dbName="mongodb-atlas" dbProtocol="mongodb" databaseAddress="mongodb+srv://cluster0.abcde.mongodb.net" !)
+
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+  Teleport provides Helm charts for installing the Teleport Database Service in Kubernetes Clusters.
+
+  (!docs/pages/includes/kubernetes-access/helm/helm-repo-add.mdx!)
+
+  (!docs/pages/includes/database-access/db-helm-install.mdx dbName="mongodb-atlas" dbProtocol="mongodb" databaseAddress="mongodb+srv://cluster0.abcde.mongodb.net" !)
```

Comment:
I think we should change this partial so it doesn't include tabs. Otherwise, we get a nested `Tabs` component, which is a poor user experience.